opcua/client/client.py

```diff
@@ -148,6 +151,20 @@ def set_password(self, pwd):
         """
         self._password = pwd
 
+    def set_max_messagesize(self, maxMessageSize):
+        """
+        Set the max messagesize for the connection.
+        """
+        self._maxMessageSize = maxMessageSize
+    
+
+    def set_max_chunkcount(self, maxChunkCount):
+        """
+        Set the max chunkcount for the connection.
+        """
+        self._maxChunkCount = maxChunkCount
```

Comment:
Not sure what is best practice here. Should we use set method or public properties. I think pep8 says public properties

Reply:
I tried to align to the current style of the codebase (e.g. `client.set_password(...)`). 
So it seems that get / set methods fits well.
Currently the default values of `_maxMessageSize` and `_maxChunkCount` is zero (unlimited). Besides the fact that it is a valid value according to the opc ua specification it seems that 65536 is a better value regarding the related issue. 
So the client will work with problematic servers that don't handle the 0 value by default. Not sure if this maybe is a problem for other opc ua servers that are supported by the last release without any additional configuration.
Because of some doubts I decided to set the default value to 0 so that it is absolutely the same state as before. This means in my case I always need to set the values to 65536 before for a successful connection.
I'm willing to make some more modifications but I need some input.